src/authn/ccloudStateHandling.test.ts

```diff
@@ -179,59 +177,38 @@ describe("authn/ccloudPolling.ts watchCCloudConnectionStatus()", () => {
   beforeEach(() => {
     sandbox = sinon.createSandbox();
 
-    getCCloudConnectionStub = sandbox.stub(connections, "getCCloudConnection");
     nonInvalidTokenStatusFireStub = sandbox.stub(nonInvalidTokenStatus, "fire");
 
     stubResourceManager = sinon.createStubInstance(ResourceManager);
     sandbox.stub(ResourceManager, "getInstance").returns(stubResourceManager);
-
-    pollFastStub = sandbox.stub(pollCCloudConnectionAuth, "useFastFrequency");
-    pollSlowStub = sandbox.stub(pollCCloudConnectionAuth, "useSlowFrequency");
   });
 
   afterEach(() => {
     sandbox.restore();
   });
 
-  it("should take no action when no connection is fetched from the sidecar", async () => {
-    getCCloudConnectionStub.resolves(undefined);
-
-    await watchCCloudConnectionStatus();
-
-    assert.ok(stubResourceManager.setCCloudAuthStatus.notCalled);
-    assert.ok(pollFastStub.notCalled);
-    assert.ok(nonInvalidTokenStatusFireStub.notCalled);
-    assert.ok(pollSlowStub.notCalled);
-  });
-
   const nonTransientStatuses: Status[] = ["FAILED", "NO_TOKEN", "VALID_TOKEN"];
   nonTransientStatuses.forEach((status) => {
     it(`should fire the nonInvalidTokenStatus event emitter when the CCloud auth status is ${status}`, async () => {
       const connection = createFakeConnection(120);
       connection.status.authentication.status = status;
-      getCCloudConnectionStub.resolves(connection);
 
-      await watchCCloudConnectionStatus();
+      await reactToCCloudAuthState(connection);
```

Comment:
Test renamed spelling, passing in the CCloud connection state instead of it polling it with a GET call.